src/midi_Settings.h

```diff
@@ -61,17 +61,25 @@ struct DefaultSettings
     */
     static const bool HandleNullVelocityNoteOnAsNoteOff = true;
 
+    /*! Active Sensing is intended to be sent
+    repeatedly by the sender to tell the receiver that a connection is alive. Use
+    of this message is optional. When initially received, the
+    receiver will expect to receive another Active Sensing
+    message each 300ms (max), and if it does not then it will
+    assume that the connection has been terminated. At
+    termination, the receiver will turn off all voices and return to
+    normal (non- active sensing) operation.
+
+    Setting this field to 0 will disable MIDI active sensing.
+    Typical value is 300 (ms) - an Active Sensing command is send every 300ms.
+    */
+    static const uint16_t SenderActiveSensingPeriodicity = 0;
+
     /*! Setting this to true will make MIDI.read parse only one byte of data for each
     call when data is available. This can speed up your application if receiving
     a lot of traffic, but might induce MIDI Thru and treatment latency.
     */
-    static const bool Use1ByteParsing = true;
-
-    /*! Override the default MIDI baudrate to transmit over USB serial, to
-    a decoding program such as Hairless MIDI (set baudrate to 115200)\n
-    http://projectgus.github.io/hairless-midiserial/
-    */
-    static const long BaudRate = 31250;
+    static const bool Use1ByteParsing = false;
```

Comment:
We should keep the BaudRate as a setting, as some sketches use it, even though there might be better transports, I'd like to minimize breaking changes. But that means the `Settings` will have to be passed as a template argument to `SerialMIDI`.

Reply:
I noticed that `mSerial.begin` was not called :-( - so added.
Also added `DefaultSerialSettings` in `serialMIDI.h`

Requires modification of the test scripts!